api/services/tasks/v1/tasks.proto

```diff
@@ -16,22 +16,22 @@ service Tasks {
 	// Create a task.
 	rpc Create(CreateTaskRequest) returns (CreateTaskResponse);
 
-	// Start a task.
-	rpc Start(StartTaskRequest) returns (google.protobuf.Empty);
+	// Start a process.
+	rpc Start(StartRequest) returns (StartResponse);
```

Comment:
I just kept the name simple when its being used by tasks and processes.  we use the same names for things like `Delete`